src/t8_forest/t8_forest.c

```diff
@@ -352,17 +353,27 @@ t8_forest_comm_global_num_elements (t8_forest_t forest)
   forest->global_num_elements = global_num_el;
 }
 
+
```

Comment:
The name suggest that this function builds a new forest.
Instead it is an adaptation callback.
Please change the name to resolve the confusion.
Also comment the function.

Reply:
Done